runtime/lua/vim/_meta/lpeg.lua

```diff
@@ -5,20 +5,19 @@ error('Cannot require a meta file')
 -- (based on revision 4aded588f9531d89555566bb1de27490354b91c7)
 -- with types being renamed to include the vim namespace and with some descriptions made less verbose.
 
----@defgroup vim.lpeg
----<pre>help
----LPeg is a pattern-matching library for Lua, based on
----Parsing Expression Grammars (https://bford.info/packrat/) (PEGs).
+--- @brief <pre>help
+--- LPeg is a pattern-matching library for Lua, based on
+--- Parsing Expression Grammars (https://bford.info/packrat/) (PEGs).
 ---
----                                                                    *lua-lpeg*
----                                                            *vim.lpeg.Pattern*
----The LPeg library for parsing expression grammars is included as `vim.lpeg`
----(https://www.inf.puc-rio.br/~roberto/lpeg/).
+---                                                                     *lua-lpeg*
```

Comment:
This tag seems to have vanished? (It's used in the docs.)

Reply:
It's still there from what I can tell. `lua.txt:3014`